index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Porfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
+            <a href="#">John Doe </a>
+
+            <div id="navig">
+            <button id="hum">&#9776;</button>
+                <div id="navitems">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="#">Contact</a>
+                    <a href="#"><img alt="Mail" src="/assets/mail-icon.svg" width="20" height="30"></a>
+                </div>
+                </div>
+</header>
+<!--Toolbar-->
+<section id="toolbar">
+    <div id="heytext">
+        <p id="p1">Hey There.</p>
+        <p id="p2">I’m Alex</p>
+    </div>
+    <div id="whoami">
+        <p id="p3">I’am a Software Developer<p>
+        <p id="p4">I can help you build a product , feature<br>
+            or website Look through some of my<br> work and experience! If you like what you<br>
+             see and have a project you need coded,<br> 
+        don’t hestiate to contact me.</p>
+    </div>
+    <!--Social media-->
+    <div id="socialicons">
+        <a href="#"><img alt="git" src="assets/githubicon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="linkein" src="assets/linkedinicon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="angel" src="assets/angelisticon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="twitter" src="assets/twittericon.svg" width="50" height="60"></a>
+        <a href="#"><img alt="medium" src="assets/mediumicon.svg" width="50" height="60"></a>
```

Comment:
Great design @borrnelsonk 🤝

- [ ] As the previous reviewer pointed out, kindly put these social icons inside `li` tags and put them inside of a `ul` tag.